src/structures/update_fuse.jl

```diff
@@ -183,7 +182,13 @@ function find_minimum_radius_for_seats_per_row(seats_per_row, ac_base)
     (minf,xopt,ret) = NLopt.optimize(opt, xopt) #Solve optimization problem starting from global solution
 
     R = xopt[1]
-    return R
+    #Check if constraint is met
+    diff = check_seats_per_row_diff(seats_per_row, xopt, ac)
```

Comment:
how is `return 0.0` handled upstream? 

on that note, this doesn't seem to be used anywhere rn

Reply:
This function is not used in sizing. It is used outside of sizing if you want to update the fuselage diameter, for example, if you have a hydrogen tank.

Reply:
The `return 0.0` is a failure case. I will replace it with an error so that it is clearer.